### PR TITLE
.github/workflows: add remote/local attestation CI/CD on ubuntu and centos

### PR DESCRIPTION
**Pull request by hustliyilin**
Fixes: #368
Signed-off-by: Yilin Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

**Review by jiazhang0 — APPROVED** [review]

File: .github/workflows/rune_centos_with_docker_and_crictl.yml [hunk 1]
```diff
@@ -114,6 +115,69 @@ jobs:
...
+    - name: Generate skeleton ra config.json on centos
+      if: always()
+      run: |
+        docker exec $centos bash -c "sed -i '161,163d' config.json;
+        sed -i 's/\/var\/run\/rune\/liberpal-skeleton-v2.so/\/usr\/lib\/liberpal-skeleton-v3.so/g' config.json;
+        sed -i '10 c  \"sh\",\"wait_timeout\",\"6000\"' config.json;
+        sed -i '144 a \"enclave.attestation.ra_epid_is_linkable\": \"false\",' config.json;
+        sed -i '144 a \"enclave.attestation.ra_epid_subscription_key\": \"${{ secrets.SUB_KEY }}\",' config.json;
+        sed -i '144 a \"enclave.attestation.ra_epid_spid\": \"${{ secrets.SPID }}\",' config.json;
+        sed -i '144 a \"enclave.attestation.ra_type\":\"EPID\",' config.json;"
```
Comment (haosanzi) [inline .github/workflows/rune_centos_with_docker_and_crictl.yml:145]: Remove `;` in line 145

File: .github/workflows/rune_centos_with_docker_and_crictl.yml [hunk 2]
```diff
@@ -114,6 +115,69 @@ jobs:
       run: docker exec $centos bash -c "cd /root/samples && ./clean.sh;
         crictl run --timeout 3s golang.yaml pod.yaml && ./show.sh"
 
+    - name: Create skeleton bundle on centos
+      if: always()
+      run: |
```
Comment (haosanzi) [inline .github/workflows/rune_centos_with_docker_and_crictl.yml:120]: There is no need to add `|`.  Please check the other lines and add `|` when needed.

File: .github/workflows/rune_ubuntu_with_docker_and_crictl.yml [hunk 3]
```diff
@@ -49,6 +49,12 @@ jobs:
         make package components=rune;
         dpkg -i rune_$RUNE_VERSION-1_amd64.deb;
         popd"
+    
+    - name: Build and install rune binary
```
Comment (haosanzi) [inline .github/workflows/rune_ubuntu_with_docker_and_crictl.yml:53]: In the last step, install the rune package. This step is useless.

Comment (haosanzi) [reply .github/workflows/rune_ubuntu_with_docker_and_crictl.yml:53]: Or you can delete the `Build and install rune deb` step and leave the `Build and install rune binary` step.
Change the name of `Build and install rune binary` to `Build and install binaries on ubuntu`.  In addition, do the same thing on rune_centos.yml.

File: .github/workflows/rune_ubuntu_with_docker_and_crictl.yml [hunk 4]
```diff
@@ -132,6 +138,69 @@ jobs:
...
+    - name: Get remote report with rune attest command with skeleton on ubuntu
+      if: always()
+      run: |
+        docker exec $ubuntu bash -c "sleep 10;
+        rune --debug attest --isRA --linkable=false --spid=${{ secrets.SPID }} --subscription-key=${{ secrets.SUB_KEY }} skeleton-enclave-container;"
+
+    - name: Get local report with rune attest command with skeleton on ubuntu
+      if: always()
+      run: |
+        docker exec $ubuntu bash -c "rune --debug attest --reportFile=/report.bin skeleton-enclave-container;"
```
Comment (haosanzi) [inline .github/workflows/rune_ubuntu_with_docker_and_crictl.yml:184]: The last command in  docker exec bash -c " "  always without `;` Please check other steps.

**Pull request closed without merging**